unity-renderer/Assets/DCLServices/EmotesCatalog/EmotesCatalogService/LambdasEmotesCatalogService.cs

```diff
@@ -108,23 +108,34 @@ public async UniTask<WearableItem> RequestEmoteFromBuilderAsync(string emoteId,
         const string TEMPLATE_URL = "https://builder-api.decentraland.org/v1/items/:emoteId/";
         string url = TEMPLATE_URL.Replace(":emoteId", emoteId);
 
-        (WearableItemResponseFromBuilder response, bool success) = await lambdasService.GetFromSpecificUrl<WearableItemResponseFromBuilder>(
-            TEMPLATE_URL, url,
-            isSigned: true,
-            cancellationToken: cancellationToken);
+        try
+        {
+            (WearableItemResponseFromBuilder response, bool success) = await lambdasService.GetFromSpecificUrl<WearableItemResponseFromBuilder>(
+                TEMPLATE_URL, url,
+                isSigned: true,
+                cancellationToken: cancellationToken);
 
-        if (!success)
-            throw new Exception($"The request of wearables from builder '{emoteId}' failed!");
+            if (!success)
+            {
+                Debug.LogException(new Exception($"The request of wearables from builder '{emoteId}' failed!"));
+                return null;
+            }
 
-        WearableItem wearable = response.data.ToWearableItem(
-            "https://builder-api.decentraland.org/v1/storage/contents/",
-            assetBundlesUrl);
+            WearableItem wearable = response.data.ToWearableItem(
+                "https://builder-api.decentraland.org/v1/storage/contents/",
+                assetBundlesUrl);
 
-        if (!wearable.IsEmote()) return null;
+            if (!wearable.IsEmote()) return null;
 
-        OnEmoteReceived(wearable);
+            OnEmoteReceived(wearable);
 
-        return wearable;
+            return wearable;
+        }
+        catch (Exception e)
+        {
+            Debug.LogException(e);
+            return null;
```

Comment:
It will require to do null-checking every call. Why not handling exceptions on outer scope?

Reply:
https://github.com/decentraland/unity-renderer/blob/dev/unity-renderer/Assets/DCLServices/EmotesCatalog/EmotesCatalogService/LambdasEmotesCatalogService.cs#L123

This method already returns a null, the code that uses this method already checks for nulls.

https://github.com/decentraland/unity-renderer/blob/dev/unity-renderer/Assets/Scripts/MainScripts/DCL/AvatarSystem/AvatarCurator.cs#L94